bundles/automation/org.eclipse.smarthome.automation.parser.gson/src/main/java/org/eclipse/smarthome/automation/parser/gson/internal/ConfigurationDeserializer.java

```diff
@@ -38,16 +41,39 @@ public Configuration deserialize(JsonElement json, Type typeOfT, JsonDeserializa
         Configuration configuration = new Configuration();
         JsonObject jo = (JsonObject) json;
         for (Entry<String, JsonElement> e : jo.entrySet()) {
-            JsonPrimitive value = e.getValue().getAsJsonPrimitive();
-            if (value.isString()) {
-                configuration.put(e.getKey(), value.getAsString());
-            } else if (value.isNumber()) {
-                configuration.put(e.getKey(), value.getAsBigDecimal());
-            } else if (value.isBoolean()) {
-                configuration.put(e.getKey(), value.getAsBoolean());
+            JsonElement element = e.getValue();
+            if (element.isJsonArray()) {
+                String key = e.getKey();
+                JsonArray array = element.getAsJsonArray();
+                configuration.put(key, new ArrayList<Object>());
+                for (JsonElement arrayElement : array) {
+                    consume(key, arrayElement.getAsJsonPrimitive(), configuration);
+                }
+            } else {
+                consume(e.getKey(), element.getAsJsonPrimitive(), configuration);
             }
         }
         return configuration;
     }
 
+    @SuppressWarnings("unchecked")
+    private void consume(String key, JsonPrimitive value, Configuration configuration) {
+        Object res = null;
+        if (value.isString()) {
+           res = value.getAsString();
+        } else if (value.isNumber()) {
+            res = value.getAsBigDecimal();
+        } else if (value.isBoolean()) {
+            res = value.getAsBoolean();
+        }
+        if (res == null) return;
```

Comment:
At least this line looks like your IDE is not working correctly. The save action should trigger the formatter and the formatter should change this line.